src/Catalogue/Author.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akondas\Library\Catalogue;
+
+use Webmozart\Assert\Assert;
+
+class Author
+{
+    private string $author;
+
+    /**
+     * Author constructor.
+     */
+    public function __construct(string $author)
+    {
+        Assert::stringNotEmpty($author, 'Author cannot be empty');
```

Comment:
I am not convinced to use external dependencies for such simple assertions. I wanted the domain code to have no dependencies on external libraries - but we already have `munus` (although it's more of a data structure). Are there any reasons why it is better to use such a library for assertions? (you can convince me, this is actually an implementation detail so I won't be too insistent :P)

Reply:
I just like this method because I think it looks really clean, but you're right, this is a too simple case to use an external package here. I will change it to a simple comparison :)

Reply:
Thanks, it's great that you were convinced :wink: